util/ghidrathon_configure.py

```diff
@@ -91,7 +93,8 @@ def main(args):
     logger.debug('Using Python home located at "%s".', home_path)
 
     json_: str = json.dumps(ghidrathon_save)
-    save_path: pathlib.Path = install_path / "ghidrathon.save"
+    save_path: pathlib.Path = pathlib.Path(os.environ.get(GHIDRATHON_SAVE_PATH) or install_path) / "ghidrathon.save"
```

Comment:
Let's validate that `os.environ.get(GHIDRATHON_SAVE_PATH)` is a directory that exists and warn the user if it's not. We should avoid situations where a user makes a small typo when creating their environment variable and the script silently ignores the error and uses `install_path` instead.

Reply:
Yeah, it makes sense to avoid ambiguity. I implemented this in cd9c8216f67e38ab3f2549331e03f7353aeeb4f8